golang/README.md

```diff
@@ -0,0 +1,96 @@
+# gotvm - Golang Frontend for TVM Runtime
+
+This folder contain golang interface for TVM runtime. It brings TVM runtime to Golang.
+
+- It enable c runtime api of tvm exposed to golang.
+- It enables module loading (lib, graph and params) and inference operations.
+
+## Installation
+
+### Requirements
+
+- go compiler (https://golang.org/) version 0.10 or above.
```

Comment:
Do we have pre-built images with go support? If not, I think it would be nice to mention that users may include `docker/install/ubuntu_install_golang.sh` rule in their Dockerfiles. I tested that way, it worked, but I had to `export PATH="/usr/lib/go-1.10/bin:$PATH"`. Could you please  mention that too?

Reply:
Demo CPU docker should support golang if it is build from Dockerfile.ci_cpu. cc @tqchen can confirm on this.

Anyway I added a section ```Docker``` to golang README with above info.